src/ClassApplication/Models/Classroom.cs

```diff
@@ -8,9 +8,15 @@ public class Classroom
         public int classID { get; set; }                    //From the user
         public string Name { get; set; }                    //From the user
         public string Description { get; set; }             //From the user
-        public IEnumerable<Note> Notes { get; set; }        //Pointer to other collection - Notes
+
+        // public IEnumerable<Note> Notes { get; set; }        //Pointer to other collection - Notes
         public IEnumerable<string> Filters { get; set; }    //changed from Tag //Pointer to other collection - Catalog
-        public IEnumerable<Student> Students { get; set; }  //Pointer to other collection - Students
+                                                            //public IEnumerable<Student> Students { get; set; }  //Pointer to other collection - Students
+
+        //         public IEnumerable<Note> Notes { get; set; }        //Pointer to other collection - Notes
```

Comment:
I don’t really like commented out code in master. Please fix this feature. What i would suggest is that you save the id’s of the notes and students instead of the actual students and notes themselves. So instead of `IEnumerable<Student>` it would be `IEnumerable<string>`